moja-kuhinjica-frontend/src/components/slider/MySlider.tsx

```diff
@@ -1,82 +1,59 @@
-import React, { Component } from "react";
-import Slider from "react-slick";
-import RestaurantCard from "../restaurant/restaurant_card/RestaurantCard";
-import "slick-carousel/slick/slick.css"
-import "slick-carousel/slick/slick-theme.css"
+import React, { Component } from 'react'
+import Slider from 'react-slick'
+import RestaurantCard from '../restaurant/restaurant_card/RestaurantCard'
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
 
 const MySlider = () => {
-var settings = {
-      dots: true,
-      infinite: true,
-      speed: 500,
-      slidesToShow: 2,
-      slidesToScroll: 1,
-      initialSlide: 0,
-      arrows: true,
-      responsive: [
-        {
-          breakpoint: 1024,
-          settings: {
-            slidesToShow: 3,
-            slidesToScroll: 3,
-            infinite: true,
-            dots: true
-          }
-        },
-        {
-          breakpoint: 600,
-          settings: {
-            slidesToShow: 2,
-            slidesToScroll: 2,
-            initialSlide: 2
-          }
-        },
-        {
-          breakpoint: 480,
-          settings: {
-            slidesToShow: 1,
-            slidesToScroll: 1
-          }
-        }
-      ]
-    };
+    var settings = {
```

Comment:
Please use `const` instead of `var`

Reply:
Also you cold maybe put this `settings` variable in seperate file

Reply:
Done.